modelplugin/aether-1.0.0/aether_1_0_0/generated.go

```diff
@@ -1,11 +1,10 @@
-// Code generated by GENERATOR. DO NOT EDIT.
```

Comment:
I had to add this line in by hand to keep the license checker happy - there should be a workaround, but requires a bit of work in the license checker

Reply:
Updated